### PR TITLE
[ADD] website_sale_product_detail_attribute_value_image

### PR DESCRIPTION
**Pull request by Tardo**
Similar to ```website_sale_product_detail_attribute_image```

attr_values_img

cc @tecnativa TT20980

### CODE REVIEW

**Review by chienandalu — CHANGES_REQUESTED** [review]
I had to struggle to find out why this was showing up like this, but it seems that if the product attribute has only one value, it renders like this:

image

File: website_sale_product_detail_attribute_value_image/README.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+.. contents::
+   :local:
+
+Usage
+=====
+
+* Go to 'Website > Settings > Configuration > Products > Attribute Values'.
+* Set an image in 'Website image' field to display this attribute in shop
+  product detail.
+* You can set alternative name for attribute in field "Website name".
```
Comment (sergio-teruel) [inline website_sale_product_detail_attribute_value_image/README.rst]: Where can I do that? I think you must remove it.

File: website_sale_product_detail_attribute_value_image/README.rst [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+* Tecnativa
+
+Contributors
+~~~~~~~~~~~~
+
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+    * Alexandre D. Díaz
+    * Sergio Teruel
```
Comment (sergio-teruel) [inline website_sale_product_detail_attribute_value_image/README.rst:75]: Uuuups!!! I do anything here?? :smile:

File: website_sale_product_detail_attribute_value_image/models/product_attribute_value.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+class ProductAttributeValue(models.Model):
+    _inherit = 'product.attribute.value'
+
+    website_product_detail_image = fields.Binary(
+        string='Website detail image',
+        attachment=True,
+        help='Image of the attribute value for shop online product detail.',
+    )
+    website_product_detail_image_published = fields.Boolean(
+        string='Publish in website',
```
Comment (sergio-teruel) [inline website_sale_product_detail_attribute_value_image/models/product_attribute_value.py]: Specifies that it is for value image... It seems tha this toggle publish or not the attribute value in website...

**Review by sergio-teruel — APPROVED** [review]
Little ux comments

**Review by carlosdauden — APPROVED** [review]

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Waiting for @chienandalu's comments to be answered + suggestion

**Review by HaraldPanten — APPROVED** [review]
Functional review 👍  LGTM

File: website_sale_product_detail_attribute_value_image/README.rst (hunk 2 above)
Comment (Tardo) [reply website_sale_product_detail_attribute_value_image/README.rst:75]: Yes! i used your module as base ;)

File: website_sale_product_detail_attribute_value_image/README.rst (hunk 1 above)
Comment (Tardo) [reply website_sale_product_detail_attribute_value_image/README.rst:43]: Here:
attr_value

**Comment by Tardo** [comment]
Changes done!
The module removes the product attributes section (only shown when the product has a single value for the attribute) on the product page to display only the icons and the value name. So, this module will show all possible values.

**Comment by chienandalu** [comment]
:red_circle: travis unhappy

**Review by chienandalu — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**